webrecorder/browsertrix
# Watch crawl from crawl detail page

(https://github.com/webrecorder/browsertrix-cloud/issues/134) Watch running crawl from the crawl detail page. Shows max 3 pages in a row before creating a second row

### Manual testing
1. Run app and start a crawl. If clicking "View Crawl", verify it takes you directly to the "View Crawl" tab
2. Otherwise, click "View Crawl" in crawl detail page. Verify view area updates with image of current crawl. Verify hovering over image shows URL

### Screenshots

2 pages in parallel:

https://user-images.githubusercontent.com/4672952/156239992-d65ec530-ecab-480e-b758-e5bd532b28ca.mov

3 pages in parallel:
<img width="868" alt="Screen Shot 2022-03-01 at 12 03 02 PM" src="https://user-images.githubusercontent.com/4672952/156240299-fe90ad52-0bcd-4ceb-ad7d-3450d2906f7c.png">
